packages/driver/test/cypress/integration/commands/actions/type_spec.js

```diff
@@ -4334,7 +4333,7 @@ describe('src/cy/commands/actions/type', () => {
         cy.on('fail', (err) => {
           expect(this.logs.length).to.eq(2)
 
-          const allChars = _.keys(Keyboard.specialChars).concat(_.keys(Keyboard.modifierChars)).join(', ')
+          const allChars = _.keys(cy.devices.keyboard.specialChars).concat(_.keys(cy.devices.keyboard.modifierChars)).join(', ')
```

Comment:
Now that I see this - these should probably be constants on the class, not the instance. approving anyway.